### ISSUE TITLE
excel-export conflict with ES6

### ISSUE THREAD

**Issue by wadezha** [functionscope/Node-Excel-Export#62]
after npm install excel-export, ES6 find function cannot work!

**Comment by haresh93** [comment]
Yes excel-export depends upon collections npm which overrides the Array.prototype.find.....is there any workaround for this??

**Comment by zhangbobell** [comment]
Yes, I've also encountered with this issue, you'd better write find with for loop 😢 , or we have to wait the author update `collections` to the latest version

**Comment by somewind** [comment]
I moved this project to https://github.com/somewind/Node-Excel-Export-Next and published a package named `excel-export-next` to fix this bug.
